### PR TITLE
Reintegrate with bors

### PR DESCRIPTION
**Pull request by MarkMcCaskey**
Attempting to get bors working again to speed up CI

### CODE REVIEW

**Comment by MarkMcCaskey** [comment]
bors try

**Comment by MarkMcCaskey** [comment]
bors try

**Comment by MarkMcCaskey** [comment]
bors try

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -1,25 +1,3 @@
-on: [push]
-
-name: build
-
-env:
```
Comment (nlewycky) [inline .github/workflows/main.yaml:5]: Do you want to keep RUST_BACKTRACE=1?

Comment (MarkMcCaskey) [reply .github/workflows/main.yaml:5]: Good catch, yeah I'll add it back

**Comment by MarkMcCaskey** [comment]
This PR is more or less ready to ship but we'll need to adjust some admin settings and maybe tweak bors (need to investigate that timeout (I suspect it's the first line in bors.toml)).

I'm going to delay shipping this PR until next week

File: bors.toml [hunk 2]
```diff
@@ -1,5 +1,8 @@
 status = [
-    "wasmerio.wasmer"
+    "wasmerio.wasmer",
```
Comment (syrusakbary) [inline bors.toml:2]: I think we don't need this status check. This was the status name of tests in Azure

**Pull request merged**